eyeshade/migrations/0017_include_indexes/up.sql

```diff
@@ -0,0 +1,13 @@
+select execute($$
+
+insert into migrations (id, description) values ('0017', 'include_indexes');
+
+CREATE INDEX transactions_filter_idx
+ON transactions (from_account, to_account)
+INCLUDE (created_at, description, channel, amount, settlement_currency, settlement_amount, transaction_type);
+
+CREATE INDEX transactions_with_type_filter_idx
```

Comment:
I think we can only create this index. Postgres should be smart enough to use ignore any additional fields. 👍

Reply:
fair enough. fixed in 1dcaf97